docs/reference/components/sources/host_metrics.cue

```diff
@@ -39,7 +39,17 @@ components: sources: host_metrics: {
 		}
 		notices: []
 		requirements: []
-		warnings: []
+		warnings: [
+			"""
+			    When vector is run under kubernetes, you may experience an error when loading the partition usage data:
```

Comment:
Isn't the source installed automatically on Kubernetes via the Helm charts? Why wouldn't we add this for them?

Reply:
Mostly because I am unfamiliar with the Helm charts, and partly because I am unclear if this is a particularly common issue with Kubernetes (appears to be common enough).

Reply:
It may well be, but that's a more difficult proposition than simply ignoring this directly (which is unlikely to produce any useful metrics).

Reply:
I updated the Helm chart as best as I could.